tests/framework/InternalParallelTests/test_internal_parallel_PP_LS.xml

```diff
@@ -11,14 +11,16 @@
     </description>
```

Comment:
(comment is for file above this one, but you don't get to comment on binaries) Why is `tests/framework/DataObjects/DataobjectsAttributes/DatabaseStorage/testGridRavenDatabase.h5` increased so much?